LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,306 @@
+    versioning) to control the interfaces,
+
+3. API Documentation for Slint and reference documentation for
+    Slint Design Markup Language, and
+
+4. Source code of Slint.
+
+## Terms and Conditions
+
+1. The Licensed Software is protected by copyright laws and international
+    copyright treaties, as well as other intellectual property laws and
+    treaties. The Licensed Software is licensed, not sold.
+
+2. The Licensed Software may provide links to third Party libraries or code
+    (collectively **Third Party Libraries**) to implement various functions.
+    Third Party Libraries do not comprise part of the Licensed Software. In some
+    cases, access to Third Party Libraries may be included along with the
+    Licensed Software delivery as a convenience for development and testing
+    only. Customer acknowledges:
+
+    1. that some Third-Party Libraries may require additional licensing of
+        copyright and/or patents from the owners of such, and
+
+    2. that distribution of any of the Licensed Software referencing any portion
+        of a Third-Party Library may require appropriate licensing from such
+        third parties. All such Third Party Libraries, along with applicable
+        copyright notices and licenses, will be listed and made available to
+        Customer.
+
+3. As part of the license grant, a Slint Software License Certificate
+    (**License Certificate**) will be issued containing details of
+
+    1. the license grant,
+
+    2. the permitted scope of Customer's use of the Licensed Software, and
+
+    3. contact details of the Customer.
+
+    In case of any changes to the information provided in 3.2 and 3.3 above,
+    Customer is obliged to notify SixtyFPS in writing, not later than 7 days
+    after any such change has occured.
+
+4. The Term of this Agreement shall begin on the date mentioned in the License
+    Certificate (**Effective Date**) and shall continue unless terminated
+    earlier as herein provided (**Term**).
+
+5. New releases of the Licensed Software (**Updates**) will be made available
+    to Customer for the period (**Maintenance Period**) as specified in the
+    License Certificate. Such Updates may be bound by additional license terms.
+    Customer consents to any such additional license terms, without which
+    Customer may not install, copy, or otherwise use such Updates.
+
+6. The Licensed Software is to be used exclusively with Customer's software
+    products (**Customer's Products**) as specified in the License
+    Certificate.
+
+7. SixtyFPS grants Customer a perpetual, non-exclusive, non-transferable right
+    to use the Licensed Software, including making modifications to the Licensed
+    Software, during the Term. The number of users using the Licensed Software
+    is limited to the amount of **User Seats** as specified in the License
+    Certificate. For clarity, each user is counted as a User Seat.
```

Comment:
In the previous license we also describe the scope: "for the sole purposes of designing, developing, and testing Customer's software". Is that a deliberate omission in this version?

Reply:
I'm still curious about this one. I thought it makes sense to clarify the scope of "use the Licensed Software" as this has come up in the past as well.

Reply:
Added back